client/state/sites/domains/assembler.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * External dependencies
+ */
+import moment from 'moment';
```

Comment:
We should avoid storing moment objects in the redux tree. We can instead decorate date objects in a selector.